src/resources/html/DaGdTag.php

```diff
@@ -49,6 +53,17 @@ protected function renderPotentialInnerTag($body) {
       return $out;
     }
 
+    if (!is_string($body)) {
+      if (is_numeric($body)) {
+        $body = (string)$body;
+      } else {
+        throw new Exception(
+          'Tag is not renderable. Body must be: null, DaGdTag, something that '.
+          'implements DaGdToTagInterface, a string, a number, or an array '.
+          'containing those things)');
```

Comment:
unbalanced parens in message
```suggestion
          'Tag is not renderable. Body must be: null, DaGdTag, something that '.
          'implements DaGdToTagInterface, a string, a number, or an array '.
          '(containing those things)');
```